src/core.js

```diff
@@ -231,9 +231,10 @@ jQuery.extend( {
 		return true;
 	},
 
-	// Evaluates a script in a global context
-	globalEval: function( code, options ) {
-		DOMEval( code, { nonce: options && options.nonce } );
+	// Evaluates a script in a provided context; falls back to the global one
+	// if not specified.
+	globalEval: function( code, options, doc ) {
```

Comment:
Oh, since this adds a new API signature, I also need to write a test for `jQuery.globalEval` explicitly.

Reply:
Done.